template/src/components/app.tsx

```diff
@@ -1,21 +1,16 @@
-import { FunctionalComponent, h } from "preact";
-import { Route, Router, RouterOnChangeArgs } from "preact-router";
+import { FunctionalComponent, h } from 'preact';
+import { Route, Router } from 'preact-router';
 
-import Home from "../routes/home";
-import Profile from "../routes/profile";
-import NotFoundPage from "../routes/notfound";
-import Header from "./header";
+import Home from '../routes/home';
+import Profile from '../routes/profile';
+import NotFoundPage from '../routes/notfound';
+import Header from './header';
 
 const App: FunctionalComponent = () => {
-    let currentUrl: string;
-    const handleRoute = (e: RouterOnChangeArgs) => {
-        currentUrl = e.url;
-    };
-
     return (
         <div id="app">
             <Header />
-            <Router onChange={handleRoute}>
```

Comment:
This was recently removed from the default template, figure it should be done here too.